src/data/grammar/grammar.json

```diff
@@ -1,8 +1,5 @@
 {
     "origin": [
-        "#headline#"
```

Comment:
I disagree with this change; keeping the default origin as a one-item like `#headline#` allows the someone's config-override to confidently have a value that is the "normal" distribution even if that changes in the future. For instance the current config-override on the server is `#headline#, #headline#, #inGraphics#`. If we added a new top-level category to `#headline#`, the server would start using it without any config changes. Can't do that if all the top-level stuff is directly in `origin`, though.